src/ts/ssh/sshServerSession.ts

```diff
@@ -253,14 +253,19 @@ export class SshServerSession extends SshSession {
 		reconnectSession.reconnectedEmitter.fire();
 	}
 
-	public dispose(): void {
+	public dispose(): void;
+
+	/* @internal */
+	public dispose(error?: Error): void;
```

Comment:
Does this add something distinct from the `dispose(error?: Error)` below?

Reply:
This is how method overloads are declared in TypeScript. First all the overloads are listed with no body, then there is one implementation for all of them.